src/panoramic/cli/command.py

```diff
@@ -160,10 +162,31 @@ def _process_table(table: Dict[str, Any]):
                 identifiers = id_generator.generate(table_name)
             else:
                 identifiers = []
-            for model in scanner.scan_columns_grouped(table_filter=table_name):
+
+            columns = list(
+                scanner.scan_columns(
+                    table_filter=table_name,
+                )
+            )
+            for model in map_columns_to_model(columns):
                 model.identifiers = identifiers
                 writer.write_scanned_model(model)
                 progress_bar.write(f'Discovered model {model.model_name}')
+
+            for column in columns:
+                column_slug = column['field_map'][0]
+                try:
+                    field = map_column_to_field(
+                        column, is_identifier=generate_identifiers and column_slug in identifiers
+                    )
+                    progress_bar.write(f'Discovered field {field.slug}')
```

Comment:
This way we'll write "Discovered field id" 10 times if there are 10 columns with that name - are we ok with that? @kubamahnert

Reply:
Per our discussion - let's change it in a follow-up if we need to.